handler/src/main/java/io/netty/handler/ssl/OpenSslCertificateException.java

```diff
@@ -24,6 +27,286 @@
  * SSL Record. This only work when {@link SslProvider#OPENSSL} is used.
  */
 public final class OpenSslCertificateException extends CertificateException {
+    public static final int X509_V_OK;
```

Comment:
can we avoid duplicating these values? Can these just be local variables (if they are necessary at all)?